src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -308,7 +308,7 @@ private void waitForModalPresence() {
         waitForElementToBeClickable(closeButton);
     }
 
-    private void waitForModalToDisappear() {
+    public void waitForModalToDisappear() {
```

Comment:
To be honest I'm not really sure why this method was private in the first place. I think this should be fine.